tailwind.config.js

```diff
@@ -41,6 +44,7 @@ export default {
         mig_blue: '#336DBA',
         gray: '#9C9AA5'
      
+>>>>>>> 48979b422e03693f5529f759fa31c9a935b254a2
```

Comment:
I see here also that you attempted to solve a merge conflict and failed. Kindly take this file back to how it was before because this stops your page from being rendered.